geomesa-spark/geomesa-spark-sql/src/main/scala/org/apache/spark/sql/SQLGeometricOutputFunctions.scala

```diff
@@ -0,0 +1,77 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.apache.spark.sql
+
+import com.vividsolutions.jts.geom.Geometry
+import org.locationtech.geomesa.utils.geohash.GeoHash
+import org.locationtech.geomesa.utils.text.WKBUtils
+
+
+object SQLGeometricOutputFunctions {
+  val ST_AsBinary: Geometry => Array[Byte] = geom => WKBUtils.write(geom)
+  val ST_AsGeoJSON: Geometry => String = geom => toGeoJson(geom)
+  val ST_AsLatLonText: Geometry => String = geom => toLatLonString(geom)
+  val ST_AsText: Geometry => String = geom => geom.toText
+  val ST_GeoHash: (Geometry, Int) => String = (geom, prec) => GeoHash(geom.getInteriorPoint, prec).hash
+
+  def registerFunctions(sqlContext: SQLContext): Unit = {
+    sqlContext.udf.register("st_asBinary", ST_AsBinary)
+    sqlContext.udf.register("st_asGeoJSON", ST_AsGeoJSON)
+    sqlContext.udf.register("st_asLatLonText", ST_AsLatLonText)
+    sqlContext.udf.register("st_asText", ST_AsText)
+    sqlContext.udf.register("st_geoHash", ST_GeoHash)
+  }
+
+  private def toGeoJson(geom: Geometry): String = {
+    val geoString = geom.toText
+    val parenthesis = geoString.indexOf('(')
+    val geomType = geoString.substring(0, parenthesis).trim
+    val formattedCoords = formatCoordinates(geoString.substring(parenthesis).trim, geomType == "POINT")
+    "{\"type\":\"" + s"${formatType(geomType)}" + "\",\"coordinates\":" + s"$formattedCoords}"
```

Comment:
This seems like there should be a function in geotools which will do this. Maybe here? 

http://docs.geotools.org/latest/javadocs/org/geotools/geojson/geom/GeometryJSON.html